pkg/config/draftconfig.go

```diff
@@ -3,12 +3,21 @@ package config
 import (
 	"errors"
 	"fmt"
+	"io/fs"
 
 	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
 )
 
+const draftConfigFile = "draft.yaml"
+
 type DraftConfig struct {
+	TemplateName        string            `yaml:"templateName"`
 	DisplayName         string            `yaml:"displayName"`
+	Description         string            `yaml:"description"`
+	Type                string            `yaml:"type"`
```

Comment:
i wonder if using a type alias with enum options would be better for this. Probably not since you only use them in yaml files, you aren't really referencing them in Go

Reply:
We might use them within the CLI at some point but the majority will be downstream consumption.  I'm going to keep if string for now and we can always swap it pretty easily at a later point